bindings/localstorage/localstorage.go

```diff
@@ -40,6 +40,7 @@ const (
 
 // List of root paths that are disallowed
 var disallowedRootPaths = []string{
+	filepath.Clean("/proc"),
```

Comment:
Probably also worth including `/sys`, `/boot`, and `/root`. There may be more that we might want to consider. `/tmp`?

Reply:
For `/root`:

- Default containers run as rootless so `/root` should not be readable.
- If someone is using the debug containers and want to run as root, then accessing `/root` is accessing the home folder, so it can make sense

I'll add `/sys` and `/boot`. I think `/tmp` needs to be allowed